code/game/gamemodes/wizard/artefact.dm

```diff
@@ -838,6 +838,7 @@ GLOBAL_LIST_EMPTY(multiverse)
 				log_game("[user][user.key] made [target][target.key] say [wgw] with a voodoo doll.")
 				log_say("Wicker doll say to [target][target.key]: [wgw]", src)
 				log_admin("[user][user.key] made [target][target.key] say [wgw] with a voodoo doll.")
+				user.create_log(SAY_LOG, "forced [target] to say [wgw] through [src].")
```

Comment:
You can also include the target here as param. It's the 3rd param. That way it will show up in the log as well. Might want to make a second one as well for the target himself so there is a reference back as well.
So:
```
user.create_log(SAY, what, target)
target.create_log(SAY, made do, user)
```